nemo/collections/nlp/models/language_modeling/megatron_bert_model.py

```diff
@@ -959,3 +1031,89 @@ def on_load_checkpoint(self, checkpoint) -> None:
                 parallel_state.set_virtual_pipeline_model_parallel_rank(i)
                 self.model[i].module.load_state_dict(checkpoint[f'model{i}'], strict=True)
             parallel_state.set_virtual_pipeline_model_parallel_rank(0)
+
+    def build_transformer_config(self) -> TransformerConfig:
```

Comment:
let's factor this out as well I think there's a lot of common configs and only have bert specific ones here